web/controllers/organization_controller.ex

```diff
@@ -0,0 +1,55 @@
+defmodule CodeCorps.OrganizationController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Organization
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create, :update]
+
+  def index(conn, _params) do
+    organizations = Repo.all(Organization)
+    organizations = Repo.preload(organizations, [:slugged_route])
+    render(conn, "index.json-api", data: organizations)
```

Comment:
Question with rendering. I've personally always used the serializer directly instead of with a Phoenix view. Is there any benefit by using a view?